.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "app/www"]
+	path = app/www
+	url = git@github.com:searchanise/cs-cart.git
+	branch = se-8418
```

Comment:
First, why do we need a sub-module? Can't we just clone repository?
Second, when `se-8418` branch will be merged and deleted, this will generate an error.

Reply:
1. Submodules are needed to avoid unnecessary actions. The comparison of cases:
 1.1. Variant with clone: `git clone git@github.com:searchanise/cs-cart-docker.git && git clone git@github.com:searchanise/cs-cart.git`
  1.2. Vs variant with sbmodules: `git clone git@github.com:searchanise/cs-cart-docker.git --recursive` In addition, knowledge about the second repository is not needed.
2. Because you are already applying my pull request, I delete it now.

Reply:
No no no, sub-modules are always is pain in the ass, believe me. Bare git clone is simpler to understand and manage. On the other hand, this is just the `cscart` stuff, so I will not block this changes.

Reply:
Yes, I think too to replace the submodule later with the composer package.